src/GraphQL.Tests/Subscription/SubscriptionSchema.cs

```diff
@@ -145,7 +153,7 @@ public MessageInputType()
     {
         Field<StringGraphType>("fromId");
         Field<StringGraphType>("content");
-        Field<DateGraphType>("sentAt");
+        Field<DateTimeOffsetGraphType>("sentAt");
```

Comment:
So the serialized responses would be repeatable as they would carry timezone info which can be set to UTC.